### PR TITLE
BULK VALIDATION ms.prod fixes, etc.

### PR DESCRIPTION
**Pull request by mSarts**

### CODE REVIEW

**Comment by mSarts** [comment]
**Global effort to fix build validation errors**

@Lauragra  

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: api-reference/beta/api/rbacapplication-post-roleassignments.md [hunk 1]
```diff
@@ -190,6 +190,8 @@ Content-type: application/json
 [!INCLUDE [sample-code](../includes/snippets/go/create-unifiedroleassignment-over-administrativeunit-go-snippets.md)]
 [!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
 
+# [PowerShell](#tab/powershell)
```
Comment (Lauragra) [inline api-reference/beta/api/rbacapplication-post-roleassignments.md:193]: Globlal: Please do not add empty tabs. We will need to update these tabs to link to the include file we're adding for when an SDK snippet is not available. @andrueastman , FYI.

File: api-reference/beta/api/shares-get.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/10/2017
 title: Access shared items
 ms.localizationpriority: medium
 doc_type: apiPageType
-ms.prod: ""
+ms.prod: microsoft-teams
```
Comment (Lauragra) [inline api-reference/beta/api/shares-get.md:8]: Change this to "files".

File: api-reference/beta/api/sites-list-followed.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ author: learafa
 description: "List the sites that have been followed by the signed in user."
 title: List followed sites
 ms.localizationpriority: medium
-ms.prod: "SharePoint"
+ms.prod: sharepoint
```
Comment (Lauragra) [inline api-reference/beta/api/sites-list-followed.md:6]: Change to "sites-and-lists".

File: api-reference/beta/api/singlevaluelegacyextendedproperty-get.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: "Get singleValueLegacyExtendedProperty"
 description: "You can get a single resource instance expanded with a specific extended property, or a collection of resource instances"
 ms.localizationpriority: medium
 doc_type: apiPageType
-ms.prod: ""
+ms.prod: outlook
```
Comment (Lauragra) [inline api-reference/beta/api/singlevaluelegacyextendedproperty-get.md:6]: non-product-specific

**Pull request merged**